docs/output.md

```diff
@@ -1,25 +1,87 @@
 # nf-core/metaboigniter: Output
 
+## Table of contents
+
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Quantification](#quantification)
+* [Annotation](#annotation)
+* [Identification](#identification)
+* [Final output](#final-output)
+
 ## Introduction
 
-The pipeline is built using [Nextflow](https://www.nextflow.io/)
-and processes data using the following steps:
+nf-core/metaboigniter produces various different output files.
 
-* [Quantification](#quantification) - quantification is performed using several steps including peak picking, feature detection, aligment and linking
-* [Annotation](#annotation) - annotation is performed using CAMERA
-* [Identification](#identification) - identification is performed using 4 search engines: Metfrag, CSI:FINGERID, CFM-ID and an internal search engine
+Each process in the workflow will create a folder with the following pattern:
+(output directory)/process_(name of the process)_(if it is library or not)_(ionization mode)_(name of the process).
 
-There are many steps in the workflow and all will be written out as output in the following format:
+The default behavior of the pipeline is to write output for identification and the final results of the pipeline (see below).
 
-**Output directory: `results/[name of the step]`**
+However, one can set the following parameter that instructs the pipeline to produce outputs for all of the tools.
 
-Each process in the workflow will create a folder with the following pattern:
-(output directory)/process_(name of the process)_(if it is library or not)_(ionization mode)_(name of the process)
+```yaml
+publishDir_intermediate: true
+```
+
+> Note that because there are a large number of parameters for this pipeline, we recommend using a YAML file and supplying to the pipeline with the Nextflow option `-params-file`.
+> Alternatively, you can create a Nextflow config file and supply this with `-c`.
+
+## Quantification
+
+Quantification is performed using several steps including peak picking, feature detection, alignment, and linking.
+
+If one chooses to perform the centroiding using OpenMS, the output of the centroiding will be mzML files.
+
+If one chooses to perform the mass trace detection using OpenMS, the output of the detection will be "featureXML" files.
+
+All other modes of quantification will produce "rdata" output files. These files have been specifically designed to work across various tools in the pipeline.
+
+The general way of reading these files outside of the workflow is to load the files using R. In almost all the cases each "rdata" file contains three important variables:
```

Comment:
```suggestion
If one chooses to perform the mass trace detection using OpenMS, the output of the detection will be `featureXML` files.

All other modes of quantification will produce `rdata` output files. These files have been specifically designed to work across various tools in the pipeline.

The general way of reading these files outside of the workflow is to load the files using R. In almost all the cases each `rdata` file contains three important variables:
```

Single backticks give inline code which will be easier to read for this kind of thing